generators/client/templates/angular/webpack/_webpack.prod.js

```diff
@@ -65,6 +65,19 @@ module.exports = webpackMerge(commonConfig({ env: ENV }), {
             ],
             exclude: ['node_modules/generator-jhipster']
         },
+        {
+            test: /\.ts$/,
+            use: [
+                {
+                    loader: 'ngc-webpack',
+                    options: {
+                        disable: false,
```

Comment:
```js
/*
 * ### resourceOverride and assets
   * If you reference assets in your styles/html that are not inlined and you expect a loader (e.g. url-loader)
   * to copy them, don't use the `resourceOverride` feature as it does not support this feature at the moment.
*/
```

Since we are using styles and html that are not inlined, this feature cannot be used.